public/javascripts/SVLabel/src/SVLabel/task/TaskContainer.js

```diff
@@ -172,39 +171,19 @@ function TaskContainer (navigationModel, neighborhoodModel, streetViewService, s
         if (typeof async == "undefined") async = true;
 
         if (typeof regionId == "number") {
-
-            // First get the priorities for the street edges in the region, then store the tasks with these priorities
             $.ajax({
-                url: "/audit/getRegionStreetPriority/" + regionId,
+                url: "/tasks?regionId=" + regionId,
                 async: async,
                 type: 'get',
                 success: function (result) {
-                    self._streetEdgePriorityMap[regionId] = {};
```

Comment:
This was the ajax call Teja wrote to get the street priorities. What I did here was to look at the commit where he made the change, and brought the code back to the same state it was in before he made the modifications.